CovidAPI.ipynb

```diff
@@ -2,12 +2,343 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "# This is the notebook we will use to make COVID API calls and build the dataframe columns relevant to COVID"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv\n",
+    "import requests\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import urllib.request\n",
+    "import codecs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "base_url = 'http://coronavirusapi.com/getTimeSeries/'\n",
+    "state = [\"AL\", \"AK\", \"AZ\", \"AR\", \"CA\", \"CO\", \"CT\", \"DC\", \n",
+    "         \"DE\", \"FL\", \"GA\", \"HI\", \"ID\", \"IL\", \"IN\", \"IA\", \n",
+    "         \"KS\", \"KY\", \"LA\", \"ME\", \"MD\", \"MA\", \"MI\", \"MN\", \n",
+    "         \"MS\", \"MO\", \"MT\", \"NE\", \"NV\", \"NH\", \"NJ\", \"NM\", \n",
+    "         \"NY\", \"NC\", \"ND\", \"OH\", \"OK\", \"OR\", \"PA\", \"RI\", \n",
+    "         \"SC\", \"SD\", \"TN\", \"TX\", \"UT\", \"VT\", \"VA\", \"WA\", \n",
+    "         \"WV\", \"WI\", \"WY\"]\n",
+    "master_list = []"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Because the dataset uses seconds from Unix Epoch date, function returns datetime, from floationg point value.\n",
+    "def epochConverter(num):\n",
+    "    days = num/3600/24\n",
+    "    date = pd.to_datetime('01-01-1970') + pd.DateOffset(days=days)\n",
+    "    return date"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Outer for loop opens url and converts each call for state from bytestring to something readable.\n",
+    "for state_url in state:\n",
+    "    state_response = urllib.request.urlopen(base_url+state_url)\n",
+    "    csvfile = csv.reader(codecs.iterdecode(state_response, 'utf-8'))\n",
```

Comment:
You could change it to 
`response = requests.get(url).text
        value_list = response.split(",")`